<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaseDialog</class>
 <widget class="QDialog" name="LeaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeaseDialog</string>
  </property>
  <widget class="QComboBox" name="CarTypeFilter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
   <property name="placeholderText">
    <string>Car type</string>
   </property>
   <item>
    <property name="text">
     <string>-no cartype-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sedan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hatchback</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SUV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coupe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Convertible</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wagon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Minivan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pickup Truck</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Van</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Luxury</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sports Car</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Muscle Car</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Roadster</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Supercar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Microcar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Compact Car</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="FuelTypeFilter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
   <property name="placeholderText">
    <string>Fuel type</string>
   </property>
   <item>
    <property name="text">
     <string>-no fueltype-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Diesel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gasoline</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hybrid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plugin-Hybrid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Electric</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="FilteredCarTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>491</width>
     <height>161</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="CarFilterResults">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cars Available:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="OKCancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>164</width>
     <height>18</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCarFilter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filter car by:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CustPhoneNr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter customer phone number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CustomerPhone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCustDetails">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer details</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>231</width>
     <height>75</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="leaseFromLabel">
      <property name="text">
       <string>Lease from</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="leaseFromDateTimeEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>3</month>
        <day>12</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="leaseUntilLabel">
      <property name="text">
       <string>Lease until</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="leaseUntilDateTimeEdit">
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>9</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="LabelLeaseCost">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total cost of lease</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CostCalcPlaceholder">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>560</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPriceMinMax">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price range:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="FilteredCustomerTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>491</width>
     <height>121</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>230</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="SeatCapacityFilter">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SeatCapLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SeatCapacity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="MinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="MaxLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MinPriceFilter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="MaxPriceFilter">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OKCancel</sender>
   <signal>accepted()</signal>
   <receiver>LeaseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKCancel</sender>
   <signal>rejected()</signal>
   <receiver>LeaseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
